src/script/lib-build.sh

```diff
@@ -72,7 +72,7 @@ function discover_compiler() {
     local cxx_compiler=g++
     local c_compiler=gcc
```

Comment:
filename in commit title

Reply:
The commit title is `src/script/lib-build.sh: ...`